configure.ac

```diff
@@ -153,6 +153,10 @@ AC_CHECK_PROGS([curl], [curl])
 CTNG_SET_KCONFIG_OPTION([curl])
 AC_SUBST([curl])
 
+AC_CHECK_PROGS([cmake], [cmake])
+CTNG_SET_KCONFIG_OPTION([cmake])
+AC_SUBST([cmake])
+
```

Comment:
I think instead of this we should do something more like https://github.com/crosstool-ng/crosstool-ng/commit/4442dc87dd5a7c237e465fd873dc40928749610a

Same applies for meson and ninja but this wasn't possible at the time those were added as dependencies.